docs/source/usage/index.rst

```diff
@@ -225,6 +225,44 @@ In order to use an external provisioning service, simply attach the node to the
 | Attach Network to Node        | ``openstack esi node network attach (--network <network> | --port <port>) <node>`` |
 +-------------------------------+------------------------------------------------------------------------------------+
 
+Serial Console Access
+---------------------
+
+Ironic nodes can be accessed through configuring a web or serial interface.
+
+In order to access a node using a serial console, the admin or owner must configure the node's console interface.
+Instructions for this can be found _here. https://docs.openstack.org/ironic/latest/admin/console.html
+Once the node is properly configured, the console can be enabled and disabled as needed.
+
++-----------------+-----------------------------------------------------+
+|                 | **Actions**                                         |
++-----------------+-----------------------------------------------------+
+| Enable Console  | ``openstack baremetal node console enable <node>``  |
++---------------------------+-------------------------------------------+
```

Comment:
The formatting of the table is off, and I think it's because line 241 has the '+' in the wrong place.